src/main/java/com/solvdLaba/airport/Airport.java

```diff
@@ -1,6 +1,6 @@
 package com.solvdLaba.airport;
 
-abstract class Airport {
+public abstract class Airport {
     protected String airportName;
```

Comment:
you can change it to name, location, since those are properties of class Airport, it is obvious that that will be airportName and airoprtLocation

Reply:
done